### DISCUSSION TITLE
Is it possible to update and append chart series data?

### DISCUSSION THREAD

**Discussion by Myles-JB** [ShapeCrawler/ShapeCrawler#489] [Q&A] [answered]
Hi,
I was wondering if it would be possible to dynamically update charts using shape crawler but not sure if it's possible.

I have a template pptx file with a dummy data chart that I want to update and insert data that I generate at run time.

The number of series/bars are dynamic based on data I'm generating which is where my problem lies. I can update already existing chart series but unable to add new series/ remove already existing.

**Comment by AdamShakhabov** [comment, marked as answer]
Hi @Myles-JB,

No, at the moment there are no APIs to add/remove Chart Series.

**Comment by AdamShakhabov** [comment]
Planned to be implemented in https://github.com/ShapeCrawler/ShapeCrawler/issues/490
